doc/website/overrides/partials/footer.html

```diff
@@ -0,0 +1,97 @@
+                rel="prev">
+                <div class="md-footer-nav__button md-icon">
+                    {% include ".icons/material/arrow-left.svg" %}
+                </div>
+                <div class="md-footer-nav__title">
+                    <div class="md-ellipsis">
+                        <span class="md-footer-nav__direction">
+                            {{ lang.t("footer.previous") }}
+                        </span>
+                        {{ page.previous_page.title }}
+                    </div>
+                </div>
+            </a>
+            {% endif %}
+            {% if page.next_page %}
+            <a href="{{ page.next_page.url | url }}" class="md-footer-nav__link md-footer-nav__link--next" rel="next">
+                <div class="md-footer-nav__title">
+                    <div class="md-ellipsis">
+                        <span class="md-footer-nav__direction">
+                            {{ lang.t("footer.next") }}
+                        </span>
+                        {{ page.next_page.title }}
+                    </div>
+                </div>
+                <div class="md-footer-nav__button md-icon">
+                    {% include ".icons/material/arrow-right.svg" %}
+                </div>
+            </a>
+            {% endif %}
+        </nav>
+    </div>
+    {% endif %}
+    <div class="md-footer-meta md-typeset">
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-footer-copyright">
+                Made with
+                <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
+                    Material for MkDocs
+                </a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-footer-copyright">
+                Copyright © 2021 Eclipse Foundation<br>
+                <a href="https://www.eclipse.org/privacy/" target="_blank">Privacy policy</a><br>
+                <a href="http://www.eclipse.org/legal/termsofuse.php" target="_blank">Terms of use</a><br>
+                <a href="https://www.eclipse.org/legal/copyright/" target="_blank">Copyright</a><br>
+                <a href="https://www.eclipse.org/legal/" target="_blank">Legal</a><br>
+                <a href="https://www.eclipse.org/security/" target="_blank">Report a safety or security
+                    vulnerability</a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-footer-copyright">
+                Sponsored by<br>
+                <a href="https://www.eclipse.org/" target="_blank" rel="noopener" title="Eclipse"><img
+                        src="{{ base_url }}/images/eclipse-logo.svg" width="100px"></img></a><br>
+                <a href="https://www.bosch.com/" target="_blank" rel="noopener" title="Robert Bosch GmbH"><img
```

Comment:
@marthtz @ariexi Is the link to "bosch.com" from your side?

Reply:
Ok for me.